api/middleware/logger.go

```diff
@@ -52,7 +50,7 @@ func Logger() macaron.Handler {
 		var content strings.Builder
 		fmt.Fprintf(&content, "ts=%s", time.Now().Format(time.RFC3339Nano))
 
-		traceID, _ := extractTraceID(ctx.Req.Context())
+		traceID := util.ExtractTraceID(ctx.Req.Context())
 		if traceID != "" {
```

Comment:
does this mean that we'll log the traceID twice? once because we add the field in LogWithTraceID(), and once here?